app/lib/surface/SegmentedSprite.coffee

```diff
@@ -134,6 +134,7 @@ module.exports = class SegmentedSprite extends createjs.Container
     @lastAnimData = animData
 
     locals = {}
+    _.extend locals, @buildMovieClipShapes(animData.shapes)
```

Comment:
@shubhi1092 Your comment made me think of what happens if there is some regression in art that I didn't test.
If there is a regression it would throw an error on this method call, and potentially crash the game.
Should I wrap this in a try catch, and log a useful error? Then if there is an error the worst case situation is the shapes just don't work (which is what we're adding)

Reply:
Yep that sounds good!